### PR TITLE
Bug 1921836: Internationalize AccessTableRows

### PR DESCRIPTION
**Pull request by rebeccaalpert**
AccessTableRows didn't internationalize User, Group, or ServiceAccount even though the filter bar above it does. This could lead to a poor user experience since translated terms could differ from non-internationalized terms. I added logic to check for and internationalize the type so they're consistent.

Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1921836.

### CODE REVIEW

File: frontend/public/components/api-explorer.tsx [hunk 1]
```diff
@@ -548,6 +534,34 @@ const APIResourceAccessReview: React.FC<APIResourceTabProps> = ({
       transforms: [sortable],
     },
   ];
+
+  const getSubjectType = (type: string) => {
```
Comment (spadgett) [inline frontend/public/components/api-explorer.tsx:538]: I'd call this `getSubjectTypeLabel` to make it clear it's the display value and not the actual value from the resource.

```suggestion
  const getSubjectTypeLabel = (type: string) => {
```

**Comment by spadgett** [comment]
/override ci/prow/ceph-storage-plugin

The ceph-storage-plugin job should only have run on PRs with changes to `frontend/packages/ceph-storage-plugin`. It failed due to an unrelated CI config change. Overriding status on this PR.

**Comment by rebeccaalpert** [comment]
All set @spadgett.

**Review by spadgett — APPROVED** [review]
/lgtm

**Comment by spadgett** [comment]
I opened #7990 to fix the backend tests.

**Pull request merged**